capsules/src/hts221.rs

```diff
@@ -202,175 +206,166 @@ impl<'a> I2CClient for Hts221<'a> {
             self.humidity_client.map(|client| client.callback(0));
             return;
         }
-        match status {
-            Ok(()) => {
```

Comment:
I don't know why bother with this since status.is_err() is already checked above.

Reply:
Really it would be most idiomatic to only match on status once, rather than doing is `status.iserr() { ... return } ... whatever else`

I am guessing the reason it is written this way is because it was originally written using `ReturnCode`.

I don't think it matters that much though